Mono.Cecil.Cil/Symbols.cs

```diff
@@ -826,7 +826,7 @@ static SR.AssemblyName GetSymbolAssemblyName (SymbolKind kind)
 
 			var suffix = GetSymbolNamespace (kind);
 
-			var cecil_name = typeof (SymbolProvider).Assembly ().GetName ();
+			var cecil_name = typeof (SymbolProvider).GetTypeInfo ().Assembly.GetName ();
```

Comment:
Can you remove all the `GetTypeInfo()` in favor of the existing API in netstd2?

Reply:
Only tests target netstandard2.0. the producut binary still targets netstandard1.3. I don't think we want to move to netstandard2.0 yet for the product binary

Reply:
Why not?

Reply:
Because not everyone can move their apps to netstandard2.0. VS, for example, is on netstandard1.3 right now.

Reply:
I understand that, but do we actually have any tool that has a dependency on the netstd1.3 version of Cecil that can't move to 2.0?

Reply:
I don't know any specific one. Hard to say. I think it's better to be conservative. There is not that many of these APIs to be worth preventing people from using Cecil in their apps that can't move to 2.0 yet.

Reply:
I'd rather consider removing the net35 and net40 targets ...

Reply:
(I mean potentially in future, not in this PR).